Doc/library/gc.rst

```diff
@@ -135,10 +135,11 @@ The :mod:`gc` module provides the following functions:
    resulting referrers.  To get only currently live objects, call :func:`collect`
    before calling :func:`get_referrers`.
 
-   Care must be taken when using objects returned by :func:`get_referrers` because
-   some of them could still be under construction and hence in a temporarily
-   invalid state. Avoid using :func:`get_referrers` for any purpose other than
-   debugging.
+   .. warning::
+       Care must be taken when using objects returned by :func:`get_referrers` because
+       some of them could still be under construction and hence in a temporarily
+       invalid state. Avoid using :func:`get_referrers` for any purpose other than
+       debugging.
```

Comment:
One small nit, this doesn't make a visual difference,:
```suggestion
   .. warning::
      Care must be taken when using objects returned by :func:`get_referrers` because
      some of them could still be under construction and hence in a temporarily
      invalid state. Avoid using :func:`get_referrers` for any purpose other than
      debugging.
```